slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/InsertTest.scala

```diff
@@ -230,4 +230,90 @@ class InsertTest extends AsyncTest[JdbcTestDB] {
       }
     } yield ()).withPinnedSession
   }
+
+  // TODO DerbyProfile and PostgresProfile seems to have a problem in this case. check #2206 and #2207
```

Comment:
```suggestion
  // Regression test for https://github.com/slick/slick/issues/1627
  // TODO DerbyProfile and PostgresProfile seems to have a problem in this case, so we skip testing them.
  // See https://github.com/slick/slick/issues/2206 and https://github.com/slick/slick/issues/2207
```

(Makes it easier to get to the relevant issues.)